fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraLdp.java

```diff
@@ -192,21 +193,44 @@ public Response options() {
      * Retrieve the node profile
      *
      * @param rangeValue the range value
-     * @return triples for the specified node
+     * @return the binary or the triples for the specified node
      * @throws IOException if IO exception occurred
      */
     @GET
-    @Produces({TURTLE + ";qs=10", JSON_LD + ";qs=8",
-            N3, N3_ALT2, RDF_XML, NTRIPLES, APPLICATION_XML, TEXT_PLAIN, TURTLE_X,
-            TEXT_HTML, APPLICATION_XHTML_XML, "*/*"})
```

Comment:
Thanks for the update, @bseeger. For this round, I agree that the deeper architectural issue is out of scope. 
* Let's rename the method along the lines of `getResource`
* Roll back to a simpler commit
* Assess things from there.

Let's focus on the issue described in the ticket:
> When making a GET request with an incorrect mimeType, e.g. "Accept: application/turtle", fedora responds with a 500 status code. It should, instead, respond with a 400 code.